src/instruments/src/EWD/elements/PseudoFWC.tsx

```diff
@@ -1349,6 +1351,17 @@ const PseudoFWC: React.FC = () => {
             sysPage: -1,
             side: 'RIGHT',
         },
+        '0000690': // VHF 3 VOICE
+        {
+            flightPhaseInhib: [],
+            simVarIsActive: voiceVHF3 !== 0,
```

Comment:
This memo should only appear in certain flight phases.

```suggestion
            simVarIsActive: voiceVHF3 !== 0 && [1, 2, 6, 9, 10].includes(flightPhase),
```

Reply:
Shouldn't that go in `flightPhaseInhib`?

Reply:
> Shouldn't that go in flightPhaseInhib?

No - inhibit can be overridden by pressing recall during the inhibited flight phases (although not sure that even applies to memos).